charts/argo-templates/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: argo-templates
 description: Collection of generic Argo templates
 type: application
-version: 0.0.0-DSPDC-915
+version: 0.0.1
```

Comment:
@danxmoran Should I include the ticket name in the version bump here?

Reply:
No, that's just a hack used for publishing test versions for now.

This version is too low, though: The latest stable release of `argo-templates` was `0.0.3`, so this would go to `0.0.4`.

Reply:
(You can find the release list [here](https://github.com/broadinstitute/monster-helm/releases))

Reply:
(Make that `0.0.5`, sorry!)